### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by randiepathirage** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#51] [Outreachy 2022] [answered]
Hello everyone. I'm an outreachy applicant. I would happy to work on the project "Implement features for Manila User Interface". Hope I am not too late to start contributing to this project.

**Comment by jesutobi** [comment]
these are the names of the mentors @BatoolMM @alswajiab , for every request you make tag them to it.

**Comment by BatoolMM** [comment, marked as answer]
Hi @randiepathirage Welcome to OSCSA. 

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Some of the tasks in the issue have been addressed - you are also welcome to review open PR!

Let me know if you are still confused!

Feel free to comment under the issue, if you have any questions!
